src/engine/botlib/nav.h

```diff
@@ -172,7 +172,7 @@ struct FastLZCompressor : public dtTileCacheCompressor
 							   unsigned char *compressed, const int /*maxCompressedSize*/, int *compressedSize )
 	{
 
-		*compressedSize = fastlz_compress( ( const void *const ) buffer, bufferSize, compressed );
+		*compressedSize = fastlz_compress( ( void * ) buffer, bufferSize, compressed );
```

Comment:
`static_cast<const void*>` would be better

Reply:
I think the `const` shouldn't be required there (i.e. do you cast to `const` the buffers you pass as _src_ to `memcpy`?). Anyway, I added it back again as in this specific case we are doing a cast and we don't want to remove the const-ness as that's not our intent.